manim/mobject/svg/text_mobject.py

```diff
@@ -575,3 +578,331 @@ def change_alignment_for_a_line(self, alignment, line_no):
+        if self.t2g:
+            self.set_color_by_t2g()
+        # anti-aliasing
+        if self.height is None and self.width is None:
+            self.scale(TEXT_MOB_SCALE_FACTOR)
+
+    def remove_last_M(self, file_name: str):
+        """Internally used function. Use to format the rendered SVG files."""
+        with open(file_name, "r") as fpr:
+            content = fpr.read()
+        content = re.sub(r'Z M [^A-Za-z]*? "\/>', 'Z "/>', content)
+        with open(file_name, "w") as fpw:
+            fpw.write(content)
+
+    def find_indexes(self, word, text):
+        """Internally used function. Finds the indexes of ``text`` in ``word``."""
+        m = re.match(r"\[([0-9\-]{0,}):([0-9\-]{0,})\]", word)
+        if m:
+            start = int(m.group(1)) if m.group(1) != "" else 0
+            end = int(m.group(2)) if m.group(2) != "" else len(text)
+            start = len(text) + start if start < 0 else start
+            end = len(text) + end if end < 0 else end
+            return [(start, end)]
+        indexes = []
+        index = text.find(word)
+        while index != -1:
+            indexes.append((index, index + len(word)))
+            index = text.find(word, index + len(word))
+        return indexes
+
+    def full2short(self, config):
+        """Internally used function. Fomats some exapansion to short forms.
+        text2color -> t2c
+        text2font -> t2f
+        text2gradient -> t2g
+        text2slant -> t2s
+        text2weight -> t2w
+        """
+        for kwargs in [config, self.CONFIG]:
+            if kwargs.__contains__("text2color"):
+                kwargs["t2c"] = kwargs.pop("text2color")
+            if kwargs.__contains__("text2font"):
+                kwargs["t2f"] = kwargs.pop("text2font")
+            if kwargs.__contains__("text2gradient"):
+                kwargs["t2g"] = kwargs.pop("text2gradient")
+            if kwargs.__contains__("text2slant"):
+                kwargs["t2s"] = kwargs.pop("text2slant")
+            if kwargs.__contains__("text2weight"):
+                kwargs["t2w"] = kwargs.pop("text2weight")
+
+    def set_color_by_t2c(self, t2c=None):
+        """Internally used function. Sets colour for specified strings."""
+        t2c = t2c if t2c else self.t2c
+        for word, color in list(t2c.items()):
+            for start, end in self.find_indexes(word, self.original_text):
+                self.chars[start:end].set_color(color)
+
+    def set_color_by_t2g(self, t2g=None):
+        """Internally used function.Sets gradient colours for specified strings. Same as ``set_color_by_t2c``."""
+        t2g = t2g if t2g else self.t2g
+        for word, gradient in list(t2g.items()):
+            for start, end in self.find_indexes(word, self.original_text):
+                self.chars[start:end].set_color_by_gradient(*gradient)
+
+    def str2style(self, string):
+        """Internally used function. Converts text to Pango Understandable Styles/"""
```

Comment:
```suggestion
        """Internally used function. Converts text to Pango Understandable Styles."""
```

Reply:
Question: does Pango use the word "style" to describe these? If I recall correctly, these are called "slant".